### ISSUE TITLE
Duplicate messages when more than one "Account"

### ISSUE THREAD

**Issue by glitchphan** [Yakoo63/gtalksms#274]
When you put more than one account in the app (so you can interact with GTalkSMS from a 2nd account), and both accounts are signed in, the first account listed gets double the notifications (not responses). The second account only gets responses. In other words, notifications for both accounts may just be going to the first one.

**Comment by glitchphan** [comment]
Is there a preferred/correct method of using more than one account? I'm separating them with a pipe.

**Comment by Yakoo63** [comment]
You can click on the blue stars in the contact/buddies panel.

**Comment by glitchphan** [comment]
Right, so when I enable the blue stars on two "buddies", and an SMS comes in, the first buddy gets two messages (identical) and the second one gets none.
